core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpSamDAO.scala

```diff
@@ -44,4 +45,13 @@ class HttpSamDAO(baseSamServiceURL: String)(implicit val system: ActorSystem) ex
     }
   }
 
+  override def getStatus(): Future[SubsystemStatus] = {
+    val url = samServiceURL + "/status"
+    val pipeline = sendReceive
+    pipeline(Get(url)) map { response =>
```

Comment:
What if Sam is down and this throws an exception? I think you may want something like:
```
} recover { case NonFatal(e) => 
  SubsystemStatus(false, Option(List(e.getMessage)))
}
```

Reply:
doesn't processSubsystemResult handle this?

Reply:
Bah you're right, it does. Nevermind